apps/site/assets/ts/stop/components/amenities/access-amenities-helpers.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Alert, Facility } from "../../../__v3api";
+import Badge from "../../../components/Badge";
+
+export const availabilityMessage = (
+  brokenElevators: number,
+  totalElevators: number,
```

Comment:
question: this is used for escalators too right? I wonder if a more generic name here might be readable, maybe `facilityAlertsLength` and `facilityLength`